cmd/provenanced/cmd/genaccounts.go

```diff
@@ -503,6 +510,7 @@ enforced immediately.  An optional type flag can be provided or the default of C
 	cmd.Flags().String(flagEscrow, "", "A list of coins held by the marker account instance.  Note: Any supply not allocated to other accounts should be assigned here.")
 	cmd.Flags().BoolP(flagFinalize, "f", false, "Set the marker status to finalized.  Requires manager to be specified. (recommended)")
 	cmd.Flags().BoolP(flagActivate, "a", false, "Set the marker status to active.  Total supply constraint will be enforced as invariant.")
+	cmd.Flags().String(markercli.FlagRequiredAttributes, "", "comma delimited list of required attributes needed for a restricted marker to have send authority")
```

Comment:
When you've got a common parser function for a flag, it's usually a good idea to also have an `Add<whatever>FlagsToCmd` function.

Reply:
I just realized I should have used the new `AddNewMarkerFlags` here (and `ParseNewMarkerFlags` above) (added in #1426).

Reply:
Ok, I will refactor to use this on when I merge PR #1426.